File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int num1, int num2) {
    string answer = "";
    
    answer += my_string.substr(0, num1);
    answer += my_string[num2];
    answer += my_string.substr(num1+1, num2-num1-1);
    answer += my_string[num1];
    answer += my_string.substr(num2+1, my_string.size()-num2);
    
    return answer;
}

File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = list(map(str, my_string))

    a = str(my_string[num1])
    b = str(my_string[num2])

    answer[num1] = str(b)
    answer[num2] = str(a)
        
    return ''.join(answer)